docs/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Monitor recent activity logged actions.
+---
+
+# Recent activity log
+
+## Activity list
+
+On the Back Office, in Admin -> Activity list, you can consult what happens recently on the database.
+
+By default, the following actions are logged:
+
+- [Content](content_items.md) [create](create_edit_content_items.md#create-content-items), [publish](publish_instantly.md#publish-content-instantly), update, trash, recover, [delete](copy_move_hide_content.md#multi-file-move), delete translation, hide and reveal
+- Location create, delete, [hide](manage_locations_urls.md#hide-locations), reveal, update, move, [swap](manage_locations_urls.md#swap-locations) and subtree copy
+- Product [create](create_edit_product.md), update and delete
+- Product variant [create](work_with_product_variants.md#generate-variants), update and [delete](work_with_product_variants.md#delete-variants)
+- [Site](work_with_sites.md) [create](work_with_sites.md#create-a-website), [update](work_with_sites.md#edit-an-existing-website) and [delete](work_with_sites.md#delete-an-existing-website)
+
+By default, log entries are kept 30 days.
```

Comment:
We should note that this relies on a `ibexa:activity-log:truncate` command (or by calling `ActivityLogService::truncate()` method).

Reply:
We are in user doc so it's not really relevant here. I'll check if I have it properlly covered in dev doc.

Reply:
https://github.com/ibexa/documentation-developer/pull/2161/files#diff-39e3b03985124d185ec800d9ec576a593d8b720c654dba273908abadc2713a63R11-R32 might cover this properly.